src/layouts/footer/styles.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components'
+
+export const Footer = styled.footer`
+  padding: 40px 0;
+  color: ${(props) => props.theme.colors.white};
+  background-color: ${(props) => props.theme.colors.darkGrey};
```

Comment:
Maybe consider destructuring props to avoid repeating keyword?

```suggestion
  color: ${({ theme }) => theme.colors.white};
  background-color: ${({ theme }) => theme.colors.darkGrey};
```

Reply:
> Maybe consider destructuring props to avoid repeating keyword?

it is shorter :D
```
background-color: ${({ theme }) => theme.colors.darkGrey};
vs
background-color: ${p => p.theme.colors.darkGrey};
```

Reply:
Uz som si vsimol ze tento approach je pozuivany vseobecne v code base. Originalny suggestion je teda nerelavantny.

Reply:
Actually I would also prefer destructuring but we can agree on ether one.

Reply:
Za mne destructuring je také lepší varianta v případě styled components a mnoha jiných. Jako Lint pravidlo bych to ale nedával. 🙂